python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```diff
@@ -864,6 +864,10 @@ interface {{ ethernet_interface.name }}
 {%             endfor %}
 {%         endfor %}
 {%     endif %}
+{%     if ethernet_interface.sync_e.enable is arista.avd.defined(true) %}
+   sync-e
+      priority {{ ethernet_interface.sync_e.priority | arista.avd.default(127) }}
```

Comment:
We normally refrain from having EOS defaults in AVD. So I would just only output this if the value is given. I am more flexible on the docs.